apps/web/pages/[user]/[type].tsx

```diff
@@ -253,12 +255,25 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
       } as const;
     }
   }
+  const locations = eventType.locations as LocationObject[];
 
   const eventTypeObject = Object.assign({}, eventType, {
     metadata: (eventType.metadata || {}) as JSONObject,
     periodStartDate: eventType.periodStartDate?.toString() ?? null,
     periodEndDate: eventType.periodEndDate?.toString() ?? null,
     recurringEvent: (eventType.recurringEvent || {}) as RecurringEvent,
+    locations: locations.map((el) => {
+      // Filter out locations that are not to be displayed publicly
+      const values = Object.values(AppStoreLocationType);
```

Comment:
can we add a fallback here for when no locations set to map doesn't fail?
`locations: locations?.map((el) => {`
```suggestion
    locations: locations.map((el) => {
      // Filter out locations that are not to be displayed publicly
      const values = Object.values(AppStoreLocationType);
```